### ISSUE TITLE
Represent OperationsMaterialBill as 1st order object and OperationsSegment as target role in composite relationship from OperationsDefinition in OperationsDefinition.xsd to align with updated 950002

### ISSUE THREAD

**Issue by racheliurui** [ETPartners/b2mmlv7beta#36]
**Background and Solutions** 
ERDi, MESA: OperationsDefnition.xsd
In the Updated 950002 Operations Definition Model, Operations Definition object has an "Has associated" association relationship to Operations Material Bill object which is a first order object.  B2MML does not represent Operations Material Bill as a 1st order object.  

B2MML is missing the target element for the "has associated" relationship to the Operations Material Bill from Both "OperationsSegmentType" and "OperationsDefinitionType" in OperationsDefinition.xsd.  

Operations Definition has a "Contains" composite relationship to Operations Segment as the target role so it is not a 1st order object but master data contained in an Operations Definition; B2MML needs to delete the Operations segment transaction types. 

Above are required to align with updated 950002 Operations Definition Model.
So the following changes are required: 
1.  complexType name = "OperationsDefinitionType
ADD:   <xsd:element name = "OperationsMaterialBillID"  	type = "IdentifierType"  minOccurs = "0" maxOccurs = "unbounded"/>

2.  complexType name = "OperationsSegmentType
ADD:   <xsd:element name = "OperationsMaterialBillID"  	type = "IdentifierType"  minOccurs = "0" maxOccurs = "unbounded"/>

3.  complexType name = "OperationsDefinitionInformationType"
ADD:   <xsd:element name = "OperationsMaterialBill"  type = "OperationsMaterialBillType"  minOccurs = "0" maxOccurs = "unbounded"/>

4. Global Elements
ADD:  <xsd:element  name = "OperationsMaterialBill"  type = "OperationsMaterialBillType" />

5. Transaction Elements
ADD:  transaction elements for OperationsMaterialBillType

6.  ADD:  OperationsMaterialBill Transaction Types 

7.  DELETE:  OperationsSegment Transaction Types
Deleted Operations segment transaction types since it a composite relationship to OperationsDefinition.

**Supporting Document** 
_ISA-950002 JWG5 CDV01 version (2019 12)_
Clause 6.1.1 Operations definition model, Figure 20 – Operations definition model 
Clause 6.1.2 Operations definition, Table 145 – Operations definition relationship roles
Clause 6.1.5 Operations segment, Table 151 – Operations segment relationship roles
Clause 6.1.3 Operations material bill, Table 147 – Operations material bill relationship roles

**Impacted Types and Solution**
_B2MML-OperationsDefinition.xsd_
- OperationsDefinitionInformationType
- OperationsDefinitionType
- OperationsSegmentType
- Global Elements, xsd:element  name = "OperationsMaterialBill"  type = "OperationsMaterialBillType"
- Transaction Elements, OperationsMaterialBillType
- OperationsMaterialBill Transaction Types 
- OperationsSegment Transaction Types

**Comment by Dennis-Brandl** [comment]
I disagree, OperationsMaterialBill is not a first order object in Part 2 or in Part 5. We should not be making changes of this magnitude. The Material Bill can always be exchanged as part of an Operations Definition, even if it is just used as a container. Remember, B2MML is not the internal structure, just a way to represent exchanged information.
